release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -115,6 +115,16 @@ For more information, see xref:../security/audit-log-policy-config.adoc#configur
 
 You can now customize the URL for the internal OAuth server. For more information, see xref:../authentication/configuring-internal-oauth.adoc#customizing-the-oauth-server-url_configuring-internal-oauth[Customizing the internal OAuth server URL].
 
+[id="ocp-4-9-security-tls-profile-setting"]
+==== Additional TLS security profile setting on the API server
+
+In {product-title} 4.9, etcd is available as a control plane component.
```

Comment:
I don't think that this sentence gets across what you intend. etcd has always been a control plane component. This feature is about etcd now honoring the TLS security profile setting. I'd suggest doing something like:

```
The Kubernetes API server TLS security profile setting is now also honored by etcd.
```

(I got this wording from the 4.8 release notes entry)

Reply:
Will do.

Reply:
Thanks for that information about the separate PRs per JIRA.